src/shogun/features/RandomKitchenSinksDotFeatures.cpp

```diff
@@ -37,14 +38,12 @@ SGMatrix<float64_t> RandomKitchenSinksDotFeatures::generate_random_coefficients(
 {
 	SGVector<float64_t> vec = generate_random_parameter_vector();
 	SGMatrix<float64_t> random_params(vec.vlen, num_samples);
-	for (index_t dim=0; dim<random_params.num_rows; dim++)
-		random_params(dim, 0) = vec[dim];
+	linalg::add_col_vec(random_params,0,vec,random_params);
```

Comment:
does this do the same thing? It assigns, rather than adding?

Reply:
Since it already initiated with 0, I think adding and assigning won't make a difference here. I'm using this btw https://www.shogun-toolbox.org/api/latest/namespaceshogun_1_1linalg.html#a604e1a0ccc54502a649d7d81cc20dd10.

Reply:
but from this code, it is not obvious that they were initialized with zero. So rather assign, easier to read

Reply:
Done !